Source/Core/Map/Sector.cs

```diff
@@ -122,6 +122,17 @@ public sealed class Sector : SelectableElement, IMultiTaggedMapElement
 		public FlatVertex[] FlatVertices { get { return flatvertices; } }
 		public ReadOnlyCollection<LabelPositionInfo> Labels { get { return labels; } }
 
+		public bool Hidden {
+			get
+			{
+				return IsFlagSet("hidden");
+			}
+			set
+			{
+				SetFlag("hidden", value);
+			}
+		}
+

```

Comment:
Not a huge fan of that. Seems kind of superfluous to add a new property when `IsFlagSet` and `SetFlag` basically do the same thing